src/remote-config/remote-config-api-client.ts

```diff
@@ -330,6 +389,61 @@ export class RemoteConfigApiClient {
         'ETag header is not present in the server response.');
     }
   }
+
+  /**
+   * Checks if a given `ListVersionsOptions` object is valid. If successful, transforms the options
+   * object in place by converting `startTime` and `endTime` to RFC3339 UTC "Zulu" format, if present.
+   * 
+   * @param {ListVersionsOptions} options An options object to be validated.
+   */
+  private validateListVersionsOptions(options: ListVersionsOptions): void {
+    if (!validator.isNonNullObject(options)) {
+      throw new FirebaseRemoteConfigError(
+        'invalid-argument',
+        'ListVersionsOptions must be a non-null object.');
+    }
+    if (typeof options.pageSize !== 'undefined' && !validator.isNumber(options.pageSize)) {
+      throw new FirebaseRemoteConfigError(
+        'invalid-argument', 'pageSize must be a number.');
+    }
+    if (typeof options.pageSize !== 'undefined' && (options.pageSize < 1 || options.pageSize > 300)) {
+      throw new FirebaseRemoteConfigError(
+        'invalid-argument', 'pageSize must be a number between 1 and 300 (inclusive).');
+    }
+    if (typeof options.pageToken !== 'undefined' && !validator.isNonEmptyString(options.pageToken)) {
+      throw new FirebaseRemoteConfigError(
+        'invalid-argument', 'pageToken must be a string value.');
+    }
+    if (typeof options.endVersionNumber !== 'undefined') {
+      this.validateVersionNumber(options.endVersionNumber, 'endVersionNumber');
```

Comment:
If the version is given as a number shouldn't we convert it into string?

Reply:
The backend API responds to both numbers and strings :)
Do you think we should always convert to string here for good measure?

Reply:
> The backend API responds to both numbers and strings

🤯 

> Do you think we should always convert to string here for good measure?

Feels like that's the right thing to do, but I'm not sure. Your call :)